src/components/ShareTrajectoryModal/index.tsx

```diff
@@ -130,15 +130,18 @@ const ShareTrajectoryModal = ({
         },
         isSharable: {
```

Comment:
nit: sp: `isShareable`

Reply:
Oh cringe, there are a couple of these. Fixing now.